timeline/schema.sql

```diff
@@ -361,6 +362,23 @@ CREATE TABLE IF NOT EXISTS "review_codes" (
 	"reason" TEXT NOT NULL
 ) STRICT;
 
+-- TODO: Still WIP / might not use this...
+CREATE TABLE IF NOT EXISTS "settings" (
+	"key" TEXT PRIMARY KEY,
+	"title" TEXT,
+	"description" TEXT,
+	"type" TEXT,
+	"value",
+	"item_id" INTEGER,
+	"attribute_id" INTEGER,
+	"data_source_id" INTEGER,
+	FOREIGN KEY ("job_id") REFERENCES "jobs"("id") ON UPDATE CASCADE ON DELETE CASCADE,
+	FOREIGN KEY ("item_id") REFERENCES "items"("id") ON UPDATE CASCADE ON DELETE CASCADE,
+	FOREIGN KEY ("entity_id") REFERENCES "entities"("id") ON UPDATE CASCADE ON DELETE CASCADE,
```

Comment:
`job_id` and `entity_id` are not columns in the table 🤔

Reply:
Ohai Mohammed! You're right... still WIP, and very very tired commit :sweat_smile: